blueflood-core/src/test/java/com/rackspacecloud/blueflood/io/serializers/SerializationTest.java

```diff
@@ -135,7 +139,35 @@ public void testFullResSerializationAndDeserializationV1() throws IOException {
     }
 
     @Test
-    public void testRollupSerializationAndDeserializationV1() throws IOException {
+    public void testBasicRollupSerializationAndDeserializationV1() throws IOException {
+
+        BasicRollupSerDes serDes = new BasicRollupSerDes();
+
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        for (BasicRollup basicRollup : TO_SERIALIZE_BASIC_ROLLUP) {
+            ByteBuffer bb = serDes.serializeV1( basicRollup );
+            baos.write(Base64.encodeBase64(bb.array()));
+            baos.write("\n".getBytes());
+        }
+        baos.close();
+
+        BufferedReader reader = new BufferedReader(new InputStreamReader(new ByteArrayInputStream(baos.toByteArray())));
+        for (int i = 0; i < TO_SERIALIZE_BASIC_ROLLUP.length; i++) {
+            ByteBuffer bb = ByteBuffer.wrap(Base64.decodeBase64(reader.readLine().getBytes()));
+            BasicRollup basicRollup = Serializers.serializerFor(BasicRollup.class).fromByteBuffer(bb);
+
+            // assert all sub-metrics are equal but sum, which is 0 for V1 rollups
+            Assert.assertEquals( "average is same after serialization for V1", TO_SERIALIZE_BASIC_ROLLUP[i].getAverage(), basicRollup.getAverage() );
+            Assert.assertEquals( "count is same after serialization for V1", TO_SERIALIZE_BASIC_ROLLUP[i].getCount(), basicRollup.getCount() );
+            Assert.assertEquals( "max is same after serialization for V1", TO_SERIALIZE_BASIC_ROLLUP[i].getMaxValue(), basicRollup.getMaxValue() );
+            Assert.assertEquals( "min is same after serialization for V1", TO_SERIALIZE_BASIC_ROLLUP[i].getMinValue(), basicRollup.getMinValue() );
+            Assert.assertEquals( "var is same after serialization for V1", TO_SERIALIZE_BASIC_ROLLUP[i].getVariance(), basicRollup.getVariance() );
+            Assert.assertEquals( "sum is 0 after serialization for V1", 0, basicRollup.getSum(), EPSILON );
+        }
+    }
+
+    @Test
+    public void testBasicRollupSerializationAndDeserialization() throws IOException {
```

Comment:
@shintasmith - I added a test to verify that sum = 0 for all BasicRollups which are serialized as V1.  This should answer the comment for verifying that sum == 0 when V1 is deserialized.